pandas/tests/series/test_dtypes.py

```diff
@@ -248,3 +249,11 @@ def test_intercept_astype_object(self):
 
         result = df.values.squeeze()
         assert (result[:, 0] == expected.values).all()
+
+    def test_series_to_categorical(self):
+        series = Series(['a', 'b', 'c'])
+
+        categorical_series = Series(series, dtype='category')
+        expected = Categorical(['a', 'b', 'c'])
```

Comment:
I think you can mak this `expected = Series(['a', 'b', 'c'], dtype='category')`

and then you can use `tm.assert_series_equal(categorical_series, expected)`.